metadata/eu.betterbird.Betterbird.appdata.xml

```diff
@@ -22,12 +29,27 @@
   <url type="donation">https://www.betterbird.eu/donate/</url>
   <url type="translate">https://github.com/Betterbird/thunderbird-patches</url>
 
+  <launchable type="desktop-id">eu.betterbird.Betterbird.desktop</launchable>
+ 
+  <releases>
+    <release version="102.2.0-bb114" date="2022-08-21" />
```

Comment:
Please be aware that this will be used as the version number for the flatpak package built at flathub. Will you make sure that it is updated on every release?

Reply:
OK. I had to correct it: https://github.com/Betterbird/thunderbird-patches/commit/ab88cc5253462c327285bf8f9919a033ec886ba1

BTW, how did you create the 15, 22, 24, 28, 32, 48, 64 and 128 PNGs? The 256 is identical to the one at https://www.betterbird.eu/media/img/betterbird-256.png.

Reply:
Thanks for the fix.

Actually I took the downscaled icons from the Linux release that I downloaded from betterbird.eu (subfolder `chrome/icons/default` of the archive).

So I give the question back to you ;-)

Reply:
the is a svg, so why downscaling?

Reply:
> Actually I took the downscaled icons from the Linux release that I downloaded from betterbird.eu (subfolder `chrome/icons/default` of the archive).
> So I give the question back to you ;-)

Hmm, we did the artwork in 2021, can't remember the details now, but there would have been a consistent process.

Well, yes, the SVG somehow makes the others obsolete if everybody can consume SVG.

May I remind you of https://github.com/flathub/flathub/pull/3435#issuecomment-1229408691. I guess that will be done when you remove desktop and appdata file from your repo (if I understood this correctly).